pkg/irreparable/common.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
```

Comment:
Since this is a database, let's make the name `irreparabledb`

Reply:
done